caprover/gc-stack-deploy/src/gc_stack_deploy/example_configs/stack.example.yaml

```diff
@@ -12,6 +12,7 @@ postgres:
   # Set only when deploy: true
   database: postgres
   version: 17
+  exposePort: 15432  # the container will map to this port on the host VM
```

Comment:
Maybe snake case since the rest of the the app-specific config is? Super minor nit, though.